intersection_corners:
  - [-3.92459965698, 0.153550537880] # NE Corner
  - [-1.3309217113663154, -0.9032005043073981] # SW Corner
velodyne:
  [-3.92459965698, 0.153550537880]
